### PR TITLE
Safetensors Dependency Emergency Fix

### PR DESCRIPTION
**Pull request by cocktailpeanut**
A new version of safetensors (0.3.3) has been released but the wheels are not available and it's breaking the installation completely.

Fixing it to `safetensors==0.3.2` fixes this problem for now.

### CODE REVIEW

**Comment by comfyanonymous** [comment]
Works fine here and I can see the wheels on https://pypi.org/project/safetensors/0.3.3/#files

**Comment by cocktailpeanut** [comment]
hmm maybe it takes time to propagate? in my case it's been trying to compile with rust and failing for the past 3 hours. let me try again and report back.

**Comment by cocktailpeanut** [comment]
Just an update, still happening. I think I explained the situation incorrectly earlier. The wheels are there, but you have to force update to the latest version otherwise the requirements won't install.

Here's another person experiencing the same thing with ComfyUI right now: https://github.com/comfyanonymous/ComfyUI/issues/1309

Also this is not just about ComfyUI. Every project out there that uses safetensors has this issue right now. For example it looks like Easydiffusion has fixed this the same way https://github.com/easydiffusion/easydiffusion/issues/1517 and even audiocraft is breaking. Fixing the version to safetensors==0.3.2 fixes this for all of these projects.

**Comment by comfyanonymous** [comment]
I do: pip install safetensors

And it installs: safetensors-0.3.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl

Is this an issue specific to windows?

**Comment by cocktailpeanut** [comment]
Yes this is happening on windows.

I think if you run `pip install -U safetensors` to force update to the latest it might work, but just running the default install fails. The #1309 issue is exactly what happens.

**Comment by M1kep** [comment]
> Works fine here and I can see the wheels on https://pypi.org/project/safetensors/0.3.3/#files

The win_amd64 whl's are missing

**Comment by JeLuF** [comment]
image

Win64 wheels have just been uploaded.

**Comment by Davikar** [comment]
> image
> 
> Win64 wheels have just been uploaded.

Yes, it is working correctly for me now, so I'll close the issue.
